Chaincase.UI/ViewModels/SendViewModel.cs

```diff
@@ -404,6 +408,40 @@ public async Task<bool> SendTransaction(string password)
             return false;
         }
 
+        private static IPayjoinClient? GetPayjoinClient(string payjoinEndPoint, bool isTorEnabled, Network network, EndPoint torSocks5EndPoint)
+        {
+            if (string.IsNullOrWhiteSpace(payjoinEndPoint) ||
+                !Uri.IsWellFormedUriString(payjoinEndPoint, UriKind.Absolute))
+            {
+                return null;
+            }
+
+            var payjoinEndPointUri = new Uri(payjoinEndPoint);
+            if (payjoinEndPointUri.DnsSafeHost.EndsWith(".onion", StringComparison.OrdinalIgnoreCase) && isTorEnabled)
+            {
+                Logger.LogWarning("PayJoin server is an onion service but Tor is disabled. Ignoring...");
```

Comment:
This log indicates to me that the condition above should be `isTorEnabled` => `!isTorEnabled`

Reply:
Good catch. This has been addressed. Please mark resolved.